src/build.props

```diff
@@ -12,6 +12,9 @@
     <VersionPrefix>2.1.0</VersionPrefix>
     <VersionSuffix></VersionSuffix>
     <PackageVersionSuffix>-rtm.0</PackageVersionSuffix>
```

Comment:
i am currently removing all dependencies on these values, i have one remaining, but thought of raising the PR first to get eyes on the approach.